openxlsx_export.h": No   such file or directory

I was trying to build the library OpenXLSX when I got this error:
mkdir build
cd build
cmake ..
make install

Install the project...
-- Install configuration: ""
CMake Error at library/cmake_install.cmake:41 (file):
  file INSTALL cannot find
  "/home/user/Downloads/OpenXLSX-master/build/library/openxlsx_export.h": No
  such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:42 (include)


make: *** [Makefile:86: install] Error 1

Can anyone help me please, i'm just a beginner.

--- a/library/CMakeLists.txt
+++ b/library/CMakeLists.txt
@@ -253,7 +253,7 @@ install(
 
 # Install export header
 install(
-        FILES ${CMAKE_CURRENT_BINARY_DIR}/openxlsx_export.h
+        FILES ${CMAKE_CURRENT_BINARY_DIR}/OpenXLSX-Exports.hpp
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openxlsx
 )
 

